tests/unit/routes/routeBackbeat.js

```diff
@@ -1012,28 +1012,6 @@ describe('routeBackbeat authorization', () => {
                 const err = JSON.parse(response.end.getCall(0).args[0]);
                 assert.strictEqual(err.code, 'AccessDenied');
             });
-
-            it('should bypass policy evaluation', async () => {
```

Comment:
maybe keep this test: it should evaluate policies (but not bucket policies) even when `bypassUserBucketPolicies` (i.e. on internal routes)